saw-remote-api/python/tests/saw/test_eval.py

```diff
@@ -0,0 +1,26 @@
+from cryptol import cryptoltypes
+import saw_client as saw
+
+import unittest
+from pathlib import Path
+
+
+def cry(exp):
+    return cryptoltypes.CryptolLiteral(exp)
+
+class ProverTest(unittest.TestCase):
+
+    def test_provers(self):
+        saw.connect(reset_server=True)
+
+        if __name__ == "__main__": saw.view(saw.LogResults())
+
+        expr1 = cry('(6 : [8]) * 7')
```

Comment:
Going forward, I think it makes more sense to use `cry_f` over `cry`, since the former has fewer sharp corners to cut yourself with.